01_install_requirements.sh

```diff
@@ -62,3 +62,11 @@ fi
 curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | sudo tee /etc/yum.repos.d/yarn.repo
 curl --silent --location https://rpm.nodesource.com/setup_10.x | sudo bash -
 sudo yum install yarn nodejs -y
+
+# Install terraform
+if [ ! -f /usr/local/bin/terraform ]; then
+    curl -O https://releases.hashicorp.com/terraform/0.11.11/terraform_0.11.11_linux_amd64.zip
+    unzip terraform_0.11.11_linux_amd64.zip
```

Comment:
Could you install unzip as well? My CentOS 7 box from beaker didn't have it.

Reply:
Thanks for pointing that out

You're good with installing terraform this way vs via an RPM or "go get"? I didn't look into options

Reply:
I don't believe they produce any official RPM's, this is exactly how I installed it.

Reply:
Added unzip, thanks